src/core/PasswordGenerator.cpp

```diff
@@ -20,6 +20,7 @@
 
 #include "crypto/Random.h"
 #include <zxcvbn.h>
+//#include <QDebug>
```

Comment:
Please remove commented code lines from the PR. It makes it easier to review the changes when you only see what was actually removed and added and is also a requirement for merging.